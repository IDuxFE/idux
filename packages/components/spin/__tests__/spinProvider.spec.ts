// import { MountingOptions, VueWrapper, mount } from '@vue/test-utils'

// import { renderWork } from '@tests'

// import SpinProvider from '../src/SpinProvider'
// import { SpinProviderInstance, SpinProviderProps } from '../src/types'

describe.skip('SpinProvider', () => {})
