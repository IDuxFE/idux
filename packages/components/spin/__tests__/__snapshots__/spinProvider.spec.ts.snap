// Vitest Snapshot v1

exports[`SpinProvider > basic > render work 1`] = `
"<div class=\\"ix-spin ix-spin-provider\\">
  <!---->
  <div class=\\"ix-spin-container\\">This is a Spin</div>
</div>"
`;
